force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +253,12 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<Integer> integerList = new List<Integer>();
+        integerList.add(1);
+        integerList.add(2);
+        integerList.add(3);
```

Comment:
Try to declare the list in the constructor
created https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex